apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -667,6 +668,9 @@ suite('thread_list_ui', function() {
 
       assert.isFalse(firstThreadNode.classList.contains('unread'));
       assert.isFalse(secondThreadNode.classList.contains('unread'));
+
+      sinon.assert.neverCalledWith(MessageManager.markThreadRead, 1, true);
```

Comment:
Why u checked that?
This seems fine:   sinon.assert.neverCalledWith(MessageManager.markThreadRead, 1, true)
Did this pass?
Also along with this , u can add the call count of that method. To check the no. Of times that method is called. I don't remember the exact syntax. Search for count , you should get the syntax. Or check sinonjs docs. Website. 
Sorry, reply from phone, so cannot provide the links now.